stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/channel/ChannelListViewModel.kt

```diff
@@ -110,6 +115,8 @@ public class ChannelListViewModel(
         return channelMutes.value.any { cid == it.channel.cid }
     }
 
+    private lateinit var queryChannelsState: QueryChannelsState
```

Comment:
We shouldn't be using lateinit vars anywhere if possible - rather use a nullable value for safety